install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -314,6 +314,24 @@ rules:
       - patch
       - delete
 
+  - apiGroups:
```

Comment:
needed for the getting the controller ref for the recorder for the feature gate accessor...